### PR TITLE
C2goto oldfrontend

### LINKED ISSUE #15: Reduce c2goto dependencies
Currently, it depends on the three old frontends, but we only use the clang frontend.

### PR DESCRIPTION
**Pull request by rafaelsamenezes**
Building the old-frontend shouldn't be enabled by default. Its usage should be limited to debugging and development. This closes https://github.com/esbmc/esbmc/issues/15

- Add the ENABLE_OLD_FRONTEND option in cmake
- Enabled this option on Travis build.

### CODE REVIEW

File: src/esbmc/esbmc_parseoptions.cpp [hunk 1]
```diff
@@ -1791,15 +1794,15 @@ void esbmc_parseoptionst::help()
        " --i386-linux                 set Linux/I386 architecture\n"
        " --i386-win32                 set Windows/I386 architecture (default)\n"
 #elif __APPLE__
-       " --i386-macos                 set MACOS/I386 architecture (default)\n"
-       " --ppc-macos                  set PPC/I386 architecture\n"
-       " --i386-linux                 set Linux/I386 architecture\n"
-       " --i386-win32                 set Windows/I386 architecture\n"
+         " --i386-macos                 set MACOS/I386 architecture (default)\n"
```
Comment (lucasccordeiro) [inline src/esbmc/esbmc_parseoptions.cpp]: why did you add these spaces?

**Review by lucasccordeiro — CHANGES_REQUESTED** [review]

File: src/esbmc/esbmc_parseoptions.cpp (hunk 1 above)
Comment (rafaelsamenezes) [reply src/esbmc/esbmc_parseoptions.cpp]: It was a mistake, not sure how it happened though

**Review by lucasccordeiro — APPROVED** [review]

**Review by mikhailramalho — APPROVED** [review]
LGTM.

However, I suggest you don't waste your time with these kind of changes, since we are planning to remove the old frontends in the future anyway.
